lib/style-transform.js

```diff
@@ -1,156 +1,85 @@
 const stylis = require('stylis')
 
-/**
- * @type {RegExp}
- *
- * matches :global($1) capturing $1
- */
-const regExpReplaceGlobal = /:global\((.*)\)/g
+function disableNestingPlugin(...args) {
+  let [context, , , parent = [], line, column] = args
```

Comment:
Oh, by named arguments i meant `(context, content, selector, parent, line, column)` not `({parent, line, column, context})`